mlflow/store/tracking/abstract_store.py

```diff
@@ -252,6 +252,19 @@ def create_trace(self, experiment_id, start_time, end_time, status, attributes,
         """
         pass
 
+    @abstractmethod
```

Comment:
This is a breaking change for third-party tracking stores :)

Reply:
Can we throw `NotImplemented` instead?

Reply:
yes, good call!